apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/InpatientClaimTransformer.java

```diff
@@ -188,7 +188,7 @@ private static ExplanationOfBenefit transformClaim(InpatientClaim claimGroup) {
         Optional.of(claimGroup.getUtilizationDayCount()));
 
     for (Diagnosis diagnosis : extractDiagnoses(claimGroup))
-      TransformerUtils.addDiagnosisCode(eob, diagnosis);
+      TransformerUtils.addImpatientDiagnosisCode(eob, diagnosis);
```

Comment:
@cbrunefearless did you mean to  spell it as addInpatientDiagnosisCode instead of add**Impatient**DiagnosisCode? Pls correct the name of the method and its reference here to correctly reflect Inpatient (Part A) diagnosis codes.